clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1796,6 +1796,26 @@ void Clang::AddAArch64TargetArgs(const ArgList &Args,
 
   Args.addOptInFlag(CmdArgs, options::OPT_fptrauth_intrinsics,
                     options::OPT_fno_ptrauth_intrinsics);
+
```

Comment:
I'd delete blank lines for these similar `OPT_fptrauth_*` lines. Deleting extra blank lines makes the grouping (`OPT_fptrauth_*` code is together) clearer.

Reply:
Deleted, see b0f9a1985846448d01af9bacdb6e3ddb23d4588f